src/omod.ml

```diff
@@ -416,7 +416,9 @@ let pp_load_sequences ppf seqs =
 type silent = [ `Yes | `Loads | `No ]
 
 let ask_which_seq ppf max =
-  let parse s = match int_of_string (String.trim s) with
+  let parse s =
+    let str = String.map (fun x -> if x = ';' then ' ' else x) s |> String.trim in
```

Comment:
If the aim was to be able to specify `;;` at the end of input that's a bit too lenient to my taste. I think It's fine just specifying the number.